tests/functional/workloads/cnv/test_vm_snapshot_cloning_ops.py

```diff
@@ -49,11 +83,32 @@ def test_vm_clone(
         )
         vm_list = vm_objs_def + vm_objs_aggr
         log.info(f"Total VMs to process: {len(vm_list)}")
-        for index, vm_obj in enumerate(vm_list):
+        for vm_obj in vm_list:
             log.info(
                 f"Starting I/O operation on VM {vm_obj.name} using "
                 f"{file_paths[0]}..."
             )
+            # Expand PVC if `pvc_expand_before_snapshot` is True
+            pvc_obj = vm_obj.get_vm_pvc_obj()
+            new_size = 50
+            if pvc_expand_before_clone:
+                try:
+                    pvc_obj.resize_pvc(new_size=new_size, verify=True)
+                    time.sleep(30)
+                    pvc_obj = vm_obj.get_vm_pvc_obj()
+                    result = vm_obj.run_ssh_cmd(command="lsblk -o SIZE")
```

Comment:
As discussed, instead of getting the lsblk output, it would be better to read the device name and check whether the specific device has been expanded. I believe you can get the device information from the VM spec file.

Reply:
Done